package.json

```diff
@@ -272,7 +275,11 @@
     "yalc": "^1.0.0-pre.44"
   },
   "dependencies": {
-    "@snyk/code-client": "^3.7.0",
+    "@itly/plugin-iteratively-node": "^2.3.1",
+    "@itly/plugin-schema-validator": "^2.3.1",
+    "@itly/plugin-segment-node": "^2.3.1",
+    "@itly/sdk": "^2.3.1",
+    "@snyk/code-client": "^3.9.0",
```

Comment:
@michelkaporin, how about updating `snyk/code-client` in separate PR? or we need some functionality for iteratively?

Reply:
good point, i've reverted the update as it slipped through during debugging of the other code.